### ISSUE TITLE
Moderator "You don't have permission to run that command!"

### ISSUE THREAD

**Issue by livgust** [scottbucher/BirthdayBot#102]
I have the following permissions:
![image](https://user-images.githubusercontent.com/8116945/97884141-b7ba4280-1cf3-11eb-8073-b29409289c40.png)
(I'm happy to share the ones below but I doubt they're relevant)
However, when I run "bday setup" I get the message "You don't have permission to run that command!"
It's a little hard to follow the code so I can't tell exactly what permission(s) I'm expected to have to run this command. Am I required to be an Administrator? If so, could the documentation be updated to state that?

**Comment by scottbucher** [comment]
You are required to either have the ADMINISTRATOR permission or the Birthday Master role (assuming it is set) in order to use that command. If you are looking to set your birthday please use **bday set** as **bday setup** is only to set server settings

This is stated here https://birthdaybot.scottbucher.dev/commands:
![image](https://user-images.githubusercontent.com/47844014/97915503-91120100-1d1f-11eb-8df4-8d954b084d2f.png)

and here https://birthdaybot.scottbucher.dev/faq:
![image](https://user-images.githubusercontent.com/47844014/97915619-c3236300-1d1f-11eb-8ada-66bc6ee885d2.png)

**Comment by livgust** [comment]
Thanks! I was able to browse around and finally found the command
`bday update birthdayMasterRole @roleName`
